<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SFileNameFactor</class>
 <widget class="QWidget" name="SFileNameFactor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_21">
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_9">
       <property name="title">
        <string>控制</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_22">
        <item row="1" column="1">
         <widget class="QPushButton" name="produceBtn">
          <property name="text">
           <string>生成</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_10">
       <property name="title">
        <string> 输出</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_23">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_24">
          <item row="1" column="0" colspan="2">
           <widget class="SDragListWidget" name="filePathsList">
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QTextEdit" name="outPutText">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string> 拖拽文件到此</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string> 路径生成</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SDragListWidget</class>
   <extends>QListWidget</extends>
   <header>Component/UI/SDragListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
